postcode_lookup/template_sorter.py

```diff
@@ -480,6 +480,11 @@ def page_title(self):
 
             if any(is_postponed(reason) for reason in cancellation_reasons):
                 verbs.append(str(_("Postponed")))
+
+            if not verbs:
+                # We don't really know what's going on here
+                # so let's assume it's postponed.
+                verbs.append(str(_("Postponed")))
```

Comment:
Applying the same logic from https://github.com/DemocracyClub/ec-postcode-lookup-pages/blob/6c351f2a6b5209f70967010dbc5f4d7e076baf47/postcode_lookup/utils.py#L144-L147 here